libs/domain/checkout/auth/src/auth.component.ts

```diff
@@ -65,14 +64,13 @@ export class CheckoutAuthComponent extends CheckoutComponentMixin<CheckoutAuthOp
   protected renderLogin(): TemplateResult {
     return html`${asyncValue(
       this.options$,
-      (options) => html`<auth-login
+      (options) => html`<oryx-auth-login
         .heading=${i18n('checkout.authentication.login')}
-        .options="${{
-          disableRedirect: true,
-          showRememberMe: true,
-          ...(options.loginUrl && { url: options.loginUrl }),
-        }}"
-      ></auth-login>`
+        .options=${{
+          disableRedirect: !options.loginUrl,
+          ...(options.loginUrl && { redirectUrl: options.loginUrl }),
```

Comment:
It can cause an error if loginUrl is falsy -> it will try to spread loginUrl value
```suggestion
          ...(options.loginUrl ? { redirectUrl: options.loginUrl } : {}),
```